### DISCUSSION TITLE
Using overnight cheap rate question?

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [nathanmarlor/foxess_em#265] [General]
Just wanted to check i'm right in thinking that if for example my battery is relatively full at the end of the evening and does not require any charge according to the HA stats, if i was to put the dishwasher and washing machine on at 12:30 this would be consumed from the grid util 04:30 and not from the battery? (Octopus GO rates)

**Comment by nathanmarlor** [comment]
Almost - it'll consume from the battery down to a set point and then hold the SoC required for the next day. Depending on the specific calculations it may be that the set point is never reached (and no grid is used), or due to a load shedding the set point is reached quickly and the remaining consumption comes from the grid.

Does this help?

**Reply by Captain0Kangaroo** [reply]
I kind of get it so think in this situation (to put on the dishwasher and or washing machine) the +1kw option would be best so not to draw on the battery too much if the next solar day is borderline.
